api/services/ocean/spark/parameters/query/maxCreatedAt.yaml

```diff
@@ -1,7 +1,7 @@
 in: query
 name: maxCreatedAt
 required: false
-example: "2021-11-22"
+example: "2021-11-22T00:00:00.000Z"
```

Comment:
Why put the microseconds and Z in the example ?

Reply:
the `Z` is indicating the UTC timezone. If you want to specify another timezone you would need to had `[±]HH[:]MM]`

The issue is : when I'm trying that in Postman, it works with the `-` , but doesn't not the `+` 🙃 . 
I'm not too sure of our support of timezone here. Could you investigate that ?

And the microseconds don't seem mandatory to me. I doubt someone needs to be that precise when requesting a list of apps